app/lib/forem_stats_driver.rb

```diff
@@ -0,0 +1,17 @@
+class ForemStatsDriver
+  def initialize(*_args)
+    @driver = select_driver.new
+  end
+
+  delegate(:increment, :time, :gauge, to: :@driver)
+  alias count increment
```

Comment:
I don't think `#count` and `#increment` is interchangeable :thinking: . I've looked through the source ([count](https://github.com/DataDog/dogstatsd-ruby/blob/master/lib/datadog/statsd.rb#L161), [increment](https://github.com/DataDog/dogstatsd-ruby/blob/master/lib/datadog/statsd.rb#L134)). `#count` replaces, and requires 2 args while `#increment` increases requires only 1 arg.

Reply:
Heh.  I just assumed that you knew something that I didn't in your comments, and went with it. :) My error, there.  Hang on, and I will correct this.  Thanks for bearing with me!